client/src/apis/chessGameApi.js

```diff
@@ -0,0 +1,11 @@
+import Config from '../config/globalConfig';
+
+const getRating = ({ playerOneRating, playerTwoRating, winner }) => {
+  const url = `${Config.serverUrl}/get-rating?playerOneRating=${playerOneRating}&playerTwoRating=${playerTwoRating}&winner=${winner}`;
+  return fetch(url)
```

Comment:
I think @chirag-shinde has placed "axios" as a dependency.
For consistency we can use axios everywhere or drop axios and use fetch everywhere.

Reply:
Ok, will use "axios" then.